langchain_benchmarks/extraction/implementations.py

```diff
@@ -0,0 +1,67 @@
+"""Default implementations of LLMs that can be used for extraction."""
+from typing import Type, Optional, List, Any, Dict
+
+from langchain.chains.openai_functions import convert_to_openai_function
+from langchain.chat_models import ChatOpenAI
+from langchain.output_parsers.openai_functions import JsonOutputFunctionsParser
+from langchain.schema.runnable import Runnable
+from langsmith.client import Client
+from pydantic import BaseModel
+
+from langchain_benchmarks.extraction.evaluators import get_eval_config
+from langchain_benchmarks.schema import ExtractionTask
+
+# PUBLIC API
+
+def create_openai_function_based_extractor(
```

Comment:
ooc: why are these diff than LangChain?

Reply:
what are they in langchain? I just cargo culted some code?

Reply:
well there was create_extraction_chain but i guess we no longer use it

Reply:
will review right now if there's a way to re-use it